src/senaite/core/upgrade/v02_07_000.zcml

```diff
@@ -2,6 +2,14 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup">
 
+  <genericsetup:upgradeStep
+      title="SENAITE.CORE 2.7.0: Migrate Worksheet objects to Dexterity"
+      description="Convert Worksheet content type from Archetypes to Dexterity"
+      source="2715"
+      destination="2716"
+      handler=".v02_07_000.migrate_worksheets_to_dx"
+      profile="senaite.core:default"/>
+
```

Comment:
Remember to add another upgrade step (or maybe in the same) to _port_ the values of the settings related with worksheets from `bikla_setup` to `senaitesetup`.